### DISCUSSION TITLE
https://zongzi531.com/2024/05/01/%E6%8A%80%E6%9C%AF%E6%85%A2%E5%A2%9E%E9%95%BF/

### DISCUSSION THREAD

**Discussion by giscus** [zongzi531/zongzi531.github.io#62] [General]
# https://zongzi531.com/2024/05/01/%E6%8A%80%E6%9C%AF%E6%85%A2%E5%A2%9E%E9%95%BF/

这个月一半是服务端和运维的身份吧，另一半回到了从前的领域，开始接触 VSCode 插件开发，也算是一个分支...

https://zongzi531.com/2024/05/01/%E6%8A%80%E6%9C%AF%E6%85%A2%E5%A2%9E%E9%95%BF/

**Comment by rebron1900** [comment]
说明状态很好。

**Reply by zongzi531** [reply]
@rebron1900 嗯，放慢心态，时间会给出答案的
